developers/verifiers/multicredentials.md

```diff
@@ -20,34 +20,70 @@ Here's how to get a demo Veres Wallet if you need one:
-3. Click "Issue Verifiable Credential" to proceed to the (optional) DID Authentication screen.
+The credential JSON below is similar to what will be passed to the issuer to generate the credential.
 
-4. Once you've completed DID Authentication (or not), you'll be provided with the Verifiable Credential JSON. Click "Store in Wallet" to proceed.
+{% include example-preissuance-credential.html %}
 
-5. You'll land at the main CHAPI Playground screen. Click "Issuer Demo".
+Now, lets issue the credential. Click "Issue Verifiable Credential" to proceed to the (optional) DID Authentication screen.
 
-Next, follow the above process a few more times to load up your wallet with example credentials.
+Once you've completed DID Authentication (or not), you'll be provided with the Verifiable Credential JSON.
 
-## Multiple VC Verification
+{% include example-issued-credential.html %}
 
-Now that you have several VCs in your wallet, let's get them all verified in one request.
+**NOTE:** The credential JSON document now has an absolute URL for it's `id` value and a new `proof` section. The `issuer.id` has also been updated to reflect the playground as the Issuer, and (of course) the `issuanceDate` has also been set.
 
-1. Visit the playground's Verifier section at <https://playground.chapi.io/verifier>.
+That credential may now be stored in a wallet. Click "Store in Wallet" to proceed.
 
-2. Click each of the VCs you loaded into your wallet (i.e. "Alumni", "JFF x vc-edu PF3", and "Permanent Resident" for example).
+A modal will appear - similar to the one below - explaining that the playground wants to send the cerdential for you.
 
-3. Click "Request Verifiable Presentation" to request that your wallet share these credentials.
+![Playground offering to store a credential. The screen contains two buttons: one to cancel the request and one to go to the next screen.](playground-offering-to-store-credential.jpg)
 
-4. In the demo Veres Wallet you'll see each of the credentials you selected (plus the ability to select which ones you want to verify). Click the "Share" button to send them to the playground for verification.
+Clicking "Next" should present you with a list of available wallets:
 
-5. The process should end with a VerifiablePresentation JSON document and the words "Presentation Verified".
+![Choose a wallet modal presenting all preregisterd wallet systems which can be clicked on to proceed to storing the credentials there.](choose-a-wallet.jpg)
+
+However, if you instead receive a browser window popup warning you that you do not yet have one set up, you will need to return to the [Wallet Sign Up section](#wallet-sign-up) above.
+
+If you received a list of wallet options, pick the one you prefer by clicking on it's name to proceed.
+
+Each wallet will have it's own system for displaying the credential and offering to store it for you.
+
+Once you've clicked "Store" in your preferred wallet system, you will be brought back to the main playground screen.
```

Comment:
in the native app flow there is potential for the app to not send them back here.. not sure if worth mentioning though

Reply:
Yeah. I'd like to make the distinction. Guess the only difference is you don't get taken back to the playground screen? So the "it worked" moment is just that it's showing in you Native App wallet?